components/admin_console/cloud_import/components/open_file.tsx

```diff
@@ -0,0 +1,362 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useRef, useState, Dispatch, SetStateAction} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import CardContainer from 'components/common/card_container';
+import {useDragAndDrop} from 'components/common/hooks/drag_and_drop';
+import FileUploadOverlay from 'components/file_upload_overlay';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import loadingIcon from 'images/cloud_spinner.gif';
+
+import ChatServiceLogoCard from '../widgets/chat_service_logo_card';
+
+import UpgradeLink from 'components/widgets/links/upgrade_link';
+
+import {
+    CloudLinks,
+} from 'utils/constants';
+
+import OpenFileImg from '../open_file.svg';
+
+import './open_file.scss';
+import FilePreview from 'components/file_preview/file_preview';
+import AlertBanner from 'components/alert_banner';
+
+export interface StepOneProps {
+    chatService: string;
+    nextStepFunc: () => void;
+    maxFreeUsers: number;
+}
+
+const StepOne: React.FC<StepOneProps> = (props: StepOneProps) => {
```

Comment:
Should this be called `OpenFile` or `OpenImportFile` given the filename `StepOne` doesn't seem to provide enough context at this component level.